client/src/app/pages/assessment/components/assessment-actions/assessment-actions-page.tsx

```diff
@@ -47,18 +49,18 @@ const AssessmentActions: React.FC = () => {
       </PageSection>
       <PageSection>
         {isArchetype ? (
-          <ConditionalRender when={!archetype} then={<AppPlaceholder />}>
+          <ConditionalRender when={!data} then={<AppPlaceholder />}>
             <TextContent>
-              {archetype ? (
-                <AssessmentActionsTable archetype={archetype} />
+              {data ? (
+                <AssessmentActionsTable archetype={data as Archetype} />
```

Comment:
If we do separate the two query hooks back out and only have one enabled, we can just pass the data from the relevant query hook here on lines 55 and 63 instead of using the `as` assertions.

Reply:
Makes sense - thanks for the review!